libs/ledger-live-common/src/exchange/swap/completeExchange.ts

```diff
@@ -229,4 +229,10 @@ const completeExchange = (
   });
 };
 
+function convertSignature(signature: string, exchangeType: ExchangeTypes): Buffer {
```

Comment:
It seems to do the same as the other convertSignature function could we do: 
convertSignature(bufferSignature: **Buffer | string** , exchangeType: ExchangeTypes): Buffer ...
and manage the two different types in the same function? 
Or I think we could rename the function to convertSignatureFromBuffer and convertSignatureFromString

Reply:
Yes, it is.
I don't want to do a rework right now as we don't have automated test for those functions.
They are both local functions for those 2 `completeExchange` functions.

I'd prefer not do to much rework as it will imply testing also SELL and FUNDS.

Reply:
understable :)